snuba/web/query.py

```diff
@@ -61,7 +62,21 @@ def parse_and_run_query(
     """
     Runs a Snuba Query, then records the metadata about each split query that was run.
     """
-    # from_clause = request.query.get_from_clause()
+
+    entity = request.query.get_from_clause()
```

Comment:
I don't know if this is the best/cleanest way to verify if a query is actually hitting generic metrics tables